src/main/kotlin/com/nylas/resources/Events.kt

```diff
@@ -80,4 +80,21 @@ class Events(client: NylasClient) : Resource<Event>(client, Event::class.java) {
     val path = String.format("v3/grants/%s/events/%s", identifier, eventId)
     return destroyResource(path, queryParams)
   }
+
+  /**
+   * Send RSVP. Allows users to respond to events they have been added to as an attendee.
+   * @param identifier The identifier of the grant to act upon
+   * @param eventId The id of the Event to update.
+   * @param requestBody The values to send the RSVP with
+   * @param queryParams The query parameters to include in the request
+   * @return The send-rsvp response
+   */
+  @Throws(NylasApiError::class, NylasSdkTimeoutError::class)
+  fun sendRsvp(identifier: String, requestBody: SendRsvpRequest, queryParams: SendRsvpQueryParams): Response<Event> {
```

Comment:
I believe the response type should be a model that represents a send rsvp response. [According to the docs](https://developer.nylas.com/docs/api/v3-beta/ecc/#post-/v3/grants/-grant_id-/events/-event_id-/send-rsvp) it should be an object like:

```
{
    request_id: string
}
```

Reply:
👍